src/Docker.DotNet/DockerClientConfiguration.cs

```diff
@@ -12,6 +12,10 @@ public class DockerClientConfiguration : IDisposable
 
         public TimeSpan DefaultTimeout { get; internal set; } = TimeSpan.FromSeconds(100);
 
+        // NamedPipeClientStream handles file not found by polling until the server arrives. Use a short
```

Comment:
Can we just have this read:
```C#
// The number of milliseconds to wait for the server to respond before the connection times out.
```
This is what the C# documentation says and its less confusing than the comment that was there before.

Reply:
Personally I did not want the comment there… I copied it from the previous location where the 100 ms was declared.
I’m inclined to remove the comment altogether if that is alright. The variable name is descriptive enough i think.

Reply:
Works for me.